src/app/page.js

```diff
@@ -1,95 +1,94 @@
-import Image from "next/image";
-import styles from "./page.module.css";
+'use client'
+import { useRef } from "react";
+import Popup from 'reactjs-popup';
 
-export default function Home() {
-  return (
-    <main className={styles.main}>
-      <div className={styles.description}>
-        <p>
-          Get started by editing&nbsp;
-          <code className={styles.code}>src/app/page.js</code>
-        </p>
-        <div>
-          <a
-            href="https://vercel.com?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            By{" "}
-            <Image
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className={styles.vercelLogo}
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
-        </div>
-      </div>
+export default function Login() {
+  const usernameRef = useRef();
+  const passwordRef = useRef();
+  const emailRef = useRef();
+
+  const blueButton = {
+    border:"solid",
+    borderRadius:"20px",
+    borderColor:"blue",
+    padding:"10px",
+    color:"white",
+    background:"blue",
+    display:"block",
+    marginRight:"auto",
+    marginLeft:"auto",
+    cursor:"pointer"
+  }
+
+  async function LoginUserWithDetails(){
+    try {
+      const response = await fetch("http://localhost:8080/login", {
+        method: "POST",
+        body: JSON.stringify(
+          {
+            Username:usernameRef.current.value,
+            Password:passwordRef.current.value
+          }),
+      });
+  
+      const result = await response.json();
+      console.log("Success:", result);
+      alert(result.message);
+    } catch (error) {
+      alert("Login attempt failed");
+      console.error("Error:", error);
+    }
+  }
 
-      <div className={styles.center}>
-        <Image
-          className={styles.logo}
-          src="/next.svg"
-          alt="Next.js Logo"
-          width={180}
-          height={37}
-          priority
-        />
-      </div>
+  function SendRecoveryEmail(){
+    alert(`Email: ${emailRef.current.value}`);
+  }
 
-      <div className={styles.grid}>
-        <a
-          href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-          className={styles.card}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <h2>
-            Docs <span>-&gt;</span>
-          </h2>
-          <p>Find in-depth information about Next.js features and API.</p>
-        </a>
+  function LoginForms(){
+    return (
+      <>
+        <form style={{textAlign:"center"}}>
```

Comment:
React Bootstrap comes with it's own HTML elements - https://react-bootstrap.netlify.app/